test/language/literals/bigint/legacy-octal-like-invalid-07n.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
```

Comment:
Is this header correct?

Reply:
I copy and past the files structure usually from the same folder. I changed this file but somehow forgot to save it. Thanks!